pkg/config/config.go

```diff
@@ -239,6 +251,10 @@ func (cd *configuration) load(cm *corev1.ConfigMap) (reconcilePolicyReport, upda
 	}
 	cd.mux.Lock()
 	defer cd.mux.Unlock()
+	//defaultRegistry, ok := cm.Data["defaultRegistry"]
+	//if !ok {
+	//	logger.V(4).Info("configuration: No defaultRegistry defined in ConfigMap")
+	//} else {
```

Comment:
Do we need this?

Reply:
My intuition behind the code (now, comment) block was that I'd need something similar for `defaultRegistry` in the method. Though, since I wasn't sure of what to add, I commented what I wrote. I'll take a look at this and remove if needed